src/event_handler/handler.py

```
@@ -1,7 +1,20 @@
-from image_event import image_event_contract_pb2
 from src.utils.read_protobuf import deserialize_proto
+from src.utils.read_protobuf import read_bytes_from_message
+from src.db import connection
+from src.db import image_repository
 
 
 def handler(event, context):
-    print(event)
+    print("upsert event handler called")
+    protobuf_message = deserialize_proto(read_bytes_from_message(event))
 
+    if protobuf_message.eventType == "UPSERT_IMAGE":
```

if the SNS splitting works correctly we shouldn't need to do this yeah?  if we do want to keep it I think we should make an exception to throw here rather than just `print`ing

---

yeah, actually we can probably just get rid of event_type in the message since we pass the event type as a message attribute on the sns and that's what it filters on